packages/clarity-js/src/data/upload.ts

```diff
@@ -43,10 +42,11 @@ export function queue(tokens: Token[], transmit: boolean = true): void {
         switch (type) {
             case Event.Discover:
             case Event.Mutation:
-                // At the moment, we limit playback to grow until MAX_PLAYBACK_BYTES. Anytime we grow past this size, we start dropping events.
+                // At the moment, we limit playback to grow until MAX_PLAYBACK_BYTES.
+                // Anytime we grow past this size, we start dropping events.
                 // This is not ideal, and more of a fail safe mechanism.
```

Comment:
nit : change comment to reflect recent changes

Reply:
Thanks, fixed.